lux/extensions/auth/models.py

```diff
@@ -29,6 +30,74 @@
 )
 
 
+class Acl(Model):
+    """
+    View-based model
+    """
+    __tablename__ = 'acl'
```

Comment:
I read in sa docs/ other sources that primary key is recommended in tables generated this way and this is the minimal unique combination. The set of records generated by the view are guaranteed to be distinct as resulting query uses grouping by all these fields. And this is exactly what we need - no duplications.